pkg/sentry/arch/arch_aarch64_unsafe.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build arm64
+
+package arch
+
+import (
+	"unsafe"
+)
+
+// initAarch64FPState sets up initial state.
+func initAarch64FPState(data *FloatingPointData) {
+	fpsimd := (*FpsimdContext)(unsafe.Pointer(data))
```

Comment:
Optional: Rather than using an unsafe cast, perhaps just use binary.LittleEndian.Uint32 to set the header? The first of the fields will be zeroed anyways.

Reply:
Done.